src/randoop/main/GenTests.java

```diff
@@ -429,264 +399,335 @@ public boolean handle(String[] args) throws RandoopTextuiException {
         }
       }
     }
+    
+    ExecutionVisitor visitor;
+    if (visitors.isEmpty()) {
+      visitor = new DummyVisitor();
+    } else {
+      visitor = new MultiVisitor(visitors);
+    }
+    
+    explorer.addTestPredicate(outputTest);
+    explorer.addTestCheckGenerator(testGen);
+    explorer.addExecutionVisitor(visitor);
 
-    explorer.executionVisitor.visitors.addAll(visitors);
-
+    if (!GenInputsAbstract.noprogressdisplay) {
+      System.out.printf ("Explorer = %s\n", explorer);
+    }
+    
+    //Generate tests
```

Comment:
? I'm not seeing what you are referring to here. The "Generate tests" comment you linked to is still there.